test/system/groups/samples_test.rb

```diff
@@ -767,5 +760,63 @@ def retrieve_puids
         click_on I18n.t('shared.samples.metadata.file_imports.errors.ok_button')
       end
     end
+
+    test 'can update metadata value that is not from an analysis' do
+      ### SETUP START ###
+      visit group_samples_url(@group)
+      assert_selector 'table thead tr th', count: 6
+
+      fill_in placeholder: I18n.t(:'groups.samples.index.search.placeholder'), with: @sample1.name
+      find('input.t-search-component').native.send_keys(:return)
+
+      assert_selector 'label', text: I18n.t('projects.samples.shared.metadata_toggle.label'), count: 1
+      find('label', text: I18n.t('projects.samples.shared.metadata_toggle.label')).click
+
+      within 'div.overflow-auto.scrollbar' do |div|
+        div.scroll_to div.find('table thead th:nth-child(7)')
+      end
+
+      assert_selector 'table thead tr th', count: 9
+      ### SETUP END ###
+
+      ### ACTIONS START ###
+      within('table tbody tr:first-child td:nth-child(7)') do
+        within('form[method="get"]') do
+          find('button').click
+        end
+        assert_selector "form[data-controller='inline-edit']"
+
+        within('form[data-controller="inline-edit"]') do
+          find('input[name="value"]').send_keys 'value2'
+          find('input[name="value"]').send_keys :return
+        end
+        ### ACTIONS END ###
+
+        ### VERIFY START ###
+        assert_no_selector "form[data-controller='inline-edit']"
+        assert_selector 'form[method="get"]'
+        assert_selector 'button', text: 'value2'
+        ### VERIFY END ###
+      end
+    end
+
+    test 'project analysts should not be able to edit samples' do
+      ### SETUP START ###
+      login_as users(:ryan_doe)
+      visit group_samples_url(@group)
+
+      find('label', text: I18n.t('projects.samples.shared.metadata_toggle.label')).click
+
```

Comment:
We should probably verify that the metadata is displayed after clicking the metadata toggle before continuing

Reply:
Added in a642d6f8109987b8a1fb09f35b18ad19bfee0b40

Reply:
Added in a642d6f8109987b8a1fb09f35b18ad19bfee0b40